num_macros/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "num_macros"
```

Comment:
Alluded to above, but this package should be called `num-macros` with our current convention of crate names on crates.io